tests/testMask.py

```diff
@@ -132,6 +132,12 @@ def testArrays(self):
         self.assertEqual(type(image3), afwImage.MaskU)
         array1[:, :] = np.random.uniform(low=0, high=10, size=array1.shape)
         self.assertMasksEqual(image1, array1)
+        array2 = image1.array
+        self.assertFloatsEqual(image1.array, array2)
```

Comment:
This is confusing because the values aren't floats. How about `numpy.testing.assert_array_equal` or `assertMasksEqual` (which does work on arrays)  instead?